src/main/webapp/app/lecture/lecture.component.html

```diff
@@ -5,19 +5,84 @@ <h2 id="page-heading">
                 <span jhiTranslate="artemisApp.lecture.home.title">Lectures</span>
             </h2>
         </div>
-        <div class="col-8 align-self-end text-end">
-            <button (click)="openImportModal()" class="btn btn-primary jh-create-entity import-lecture text-truncate mb-1" id="lecture-import-button">
-                <fa-icon [icon]="faPlus"></fa-icon>
-                <span jhiTranslate="artemisApp.lecture.home.importLabel">Import Lecture</span>
-            </button>
-            <a id="jh-create-entity" class="btn btn-primary jh-create-entity create-lecture mb-1" [routerLink]="['new']">
-                <fa-icon [icon]="faPlus"></fa-icon>
-                <span jhiTranslate="artemisApp.lecture.home.createLabel">Create Lecture</span>
-            </a>
+        <div class="d-flex flex-row-reverse bd-highlight mb-3">
```

Comment:
Since you're reusing the filter from other pages, would it be wise to move it into a common module and reuse the module in the occurrences?